### DISCUSSION TITLE
Any plan for next release?

### DISCUSSION THREAD

**Discussion by jgaulon** [dotpcap/packetnet#159] [General]
Would it be possible to get a new release of this lib?
Thanks in advance,
Julien.

**Comment by PhyxionNL** [comment]
Done! 😁

**Reply by jgaulon** [reply]
thanks!
When do you think this will be available under https://www.nuget.org/packages/PacketDotNet/ ?

**Comment by PhyxionNL** [comment]
Should be soon after latest commit.
